spec/_attachments/ic.did

```diff
@@ -111,6 +111,14 @@ type send_transaction_request = record {
 
 type millisatoshi_per_byte = nat64;
 
+type metrics = variant {
```

Comment:
```suggestion
type node_metrics = variant {
```

Reply:
Agreed with the name. Not sure if we then still need an enumeration (variant) type though.